hbase_sql/src/main/java/com/splicemachine/derby/impl/SpliceSparkKryoRegistrator.java

```diff
@@ -127,7 +127,7 @@ public void register(Kryo instance) {
 
     @Override
     public void registerClasses(Kryo instance) {
-//    	instance.setReferences(false);
+    	instance.setReferences(false);
```

Comment:
The default behavior is True. Each appearance of an object in the reference graph after the first is stored as an integer ordinal. This allows multiple references to the same object and cyclic graphs to be serialized. It has a small amount of overhead and can be disabled to save space if it is not needed. Do we have any quantifiable overhead observed in profiler? It'll help us understand the benefits better.